docs/dev/libs/cep.md

```diff
@@ -711,6 +711,57 @@ DataStream[Either[TimeoutEvent, ComplexEvent]] result = patternStream.flatSelect
 </div>
 </div>
 
+### Handling Lateness in Event Time
+
+In `CEP` the order in which elements are processed matters. To this end and when working in event time, an incoming 
```

Comment:
I don't get the first part of the sentence : "To this end and when working in event time"